cmd/create/cluster/cmd.go

```diff
@@ -579,6 +583,12 @@ func init() {
 		false,
 		"Enable the use of hosted control planes (HyperShift)",
 	)
+
+	flags.StringVar(&args.machinePoolRootDiskSize,
+		"mp-root-disk-size",
```

Comment:
I think we need to get BU sign off on this.

Personally I think we could drop the `mp-` prefix.

Reply:
I think the mp prefix makes sense considering this is the create cluster flow instead of machine pool flow

It indicates to customer a bit more clearer that the volume is for machine pool

Reply:
I thought it was as part of: https://docs.google.com/document/d/1YNBNc1peew4E4yjtoQT7YZOtvbUYImoS5ZshVFXlmX4/edit# but more comments are always welcome if you feel like changing the flag name :). Thanks!

Reply:
Let's continue the discussion in the design doc. Thanks.